### PR TITLE
Fix Comment Typo

### PR DESCRIPTION
**Pull request by ZapUser77**

### CODE REVIEW

**Comment by benthecarman** [comment]
Please squash your commits

**Comment by ZapUser77** [comment]
> 
> 
> Please squash your commits

*Looks around for a "squash button" and fails to locate one*

Sure thing, just tell me where on github is the 'squash' button.  Or precise instructions (without any command line garbage... I hate Linux and CLI) on how to do it on GitHub.

**Comment by benthecarman** [comment]
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by ZapUser77** [comment]
Sorry, but that requires CLI.  I don't actually have my fork located anywhere.  It exists strictly on Github, and I edited the file directly on Github.

File: contractcourt/chain_watcher.go [hunk 1]
```diff
@@ -308,7 +308,7 @@ func (c *chainWatcher) closeObserver(spendNtfn *chainntnfs.SpendEvent) {
 			return
 		}
 
-		// We'll not retrieve the latest sate of the revocation store
+		// We'll now retrieve the latest sate of the revocation store
```
Comment (halseth) [inline contractcourt/chain_watcher.go]: you can also fix the `s/sate/state` typo while you're at it 😄

Comment (ZapUser77) [reply contractcourt/chain_watcher.go]: Lol. There's already too many 't's in "Latest sate"  :P

**Pull request closed without merging**